### PR TITLE
feat(whitelist): Add support for ip whitlising for app

### PR DESCRIPTION
**Pull request by kmala**

### CODE REVIEW

File: cmd/whitelist.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+	quit := progress(d.WOut)
+	_, err = whitelist.Add(s.Client, appID, strings.Split(IPs, ","))
+	quit <- true
+	<-quit
+	if checkAPICompatibility(s.Client, err, d.WErr) != nil {
+		return err
+	}
+
+	d.Print("done\n")
```
Comment (Joshua-Anderson) [inline cmd/whitelist.go]: I'd do `d.Println("done")` instead.

Comment (kmala) [inline cmd/whitelist.go]: changed.

**Comment by Joshua-Anderson** [comment]
Do we want to add the whitelist to `apps:info`?

It's not a blocker or anything, just a random idea.

**Comment by kmala** [comment]
@Joshua-Anderson i think it will good to add but then we may also have many other which we can add so i will leave the final decision on the UX to @slack or @helgi

**Pull request merged**